### DISCUSSION TITLE
Any chance of getting a Boox formatted version?

### DISCUSSION THREAD

**Discussion by ehula** [kudrykv/latex-yearly-planner#109] [Q&A] [answered]
I am using the Remarkable version for now. It seems to be the best fit, but there is a wide margin on the left that is just wasted space on a Boox device. Is there any change we could get a version that fills the screen on a Boox? I have a Boox Nova 2 (soon to be getting a Boox Note 3C), but I think they have the same aspect ratio.

Also, what happened to the todos? They seemed like a good idea.

p.s. This is an excellent project. I have bought a planner, but am preferring this one. I think the one I bought is based on this project! ;)

**Comment by ehula** [comment, marked as answer]
I'd like to solve my own question.

The Remarkable version works just fine on a Boox device. In fact, I have found that the Remarkable version leaving the upper right corner open solves the problem of accidentally creating a bookmark when trying to tap the Notes button in the planner. Yes, the Remarkable version has a strip along the left edge that is either unused or in the case of the months-on-the-side, show the months, but that isn't such a big deal and can be used to display the Boox toollbar when necessary. And if you don't like the upper right corner open, you can use the DDVK version, although it does create a conflict with the Boox Neo-Reader bookmark function. (Does anyone know how to disable the bookmark button?)

All-in-all, this is a wonderful project, and I hope it continues to be maintained. Thank you @kudrykv .

**Comment by kudrykv** [comment]
> Also, what happened to the todos? They seemed like a good idea.

It was a bad decision on my part to remove them. They will get back, just not that soon as I wanted them to.

I'm glad you resolved your problem 🙂

> The Remarkable version works just fine on a Boox device

That's another thing I need to "fix". It's better to tie to screen sizes + options, not to device names. Oh well 🤷‍♀️ That's also on the to-do list

**Reply by ehula** [reply]
Glad to hear that you are planning on restoring "ToDos".

I have a request regarding ToDos and Notes. Can you make the index pages for ToDos and Notes each be a single page and contain two columns, where each ToDo/Note index is a line. I find this layout allows for more room to write a title as opposed to the layout where each index is a small box. It is very difficult to write a meaningful title into a box.
